improver_tests/wxcode/wxcode/test_WeatherSymbols.py

```diff
@@ -34,6 +34,7 @@
 
 import iris
 import numpy as np
+from datetime import datetime as dt
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [standard import "from datetime import datetime as dt" should be placed before "import iris"](https://app.codacy.com/manual/metoppv_tech/improver/pullRequest?prid=4957787)